modules/ROOT/pages/vpn-high-availability.adoc

```diff
@@ -0,0 +1,42 @@
+= Anypoint VPN High Availability
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+image:logo-cloud-active.png[xref="deployment-strategies.adoc#cloudhub-deployments", title="CloudHub"]
+image:logo-hybrid-disabled.png[xref="deployment-strategies.adoc#hybrid-deployments", title="Hybrid"]
+image:logo-server-disabled.png[xref="deployment-strategies.adoc#anypoint-platform-pce-deployments", title="Anypoint Platform PCE"]
+image:logo-rtf-disabled.png[xref="deployment-strategies.adoc#anypoint-runtime-fabric-deployments", title="Runtime Fabric"]
+
+To ensure your applications and related operations are tolerant to Anypoint VPN updates, issues, or individual Customer Gateway failures, implement High Availability VPN connections.
+
+Each Anypoint VPN connection consists of two endpoints, providing High Availability (HA) on the MuleSoft side by default.
+
+== Prerequisites
+
+* Sufficient entitlement for each VPN connection.
+* Two VPN endpoints using different public IP addresses.
+* Two VPN connections in Runtime Manager.
+
+== Configure High Availability with Anypoint VPN
+
+The example below shows a High Availability VPN topology using a single MuleSoft VPC and two VPN connections.
+
+A MuleSoft Virtual Private Gateway (VGW) can only be associated with a single MuleSoft VPC, but it supports up to 10 VPN connections.
+You can locate your VPN Gateways in the same data center, or in different physical locations.
+
+ image:vpn-ha-topology.png[VPN High Availability Topology]
+
+
+ Use BGP routing to advertise the same routes via VPN-1 and VPN-2. See https://help.mulesoft.com/s/article/Anypoint-VPN-Path-Selection-using-BGP-Routing[Anypoint VPN Path Selection using BGP Routing] for instructions on how to control path selection via the routing protocol.
+
+ In this scenario, the VPN Gateways are configured to prefer: VPN-1 Tunnel-1, then VPN-1 Tunnel-2, then VPN-2 Tunnel-1, and finally VPN-2 Tunnel-2.
+ This allows you to make use of automatic failover to another tunnel, and to another VPN in the event of a VPN connectivity issue. This makes the Anypoint VPN solution more resilient and robust.
+
+You can also configure a High Availability VPN connection using static routing. In this scenario, you can manually establish VPN-2 to work as a redundant, standby connection in the event of a failure with VPN-1. There is no automatic failover between the VPNs.
```

Comment:
Suggestion: join these sentences to make it more concise: 

High Availability VPN connections also support static routing, in which you establish a VPN-2 to work as a redundant, standby connection in the event of a failure with VPN-1.